go.mod

```diff
@@ -1,12 +1,28 @@
 module decred.org/dcrctl
 
-go 1.16
+go 1.17
 
 require (
-	decred.org/dcrwallet/v2 v2.0.7
+	decred.org/dcrwallet/v2 v2.0.10
```

Comment:
There's a v2.0.11 now

Reply:
Aye, but given master still pull in 2.0.10 and this is a backport of that, I thought it made more sense to stick with 2.0.10 instead of having a release branch ahead of master.

Reply:
I'll go ahead and bump master right quick and then backport as a part of this PR too so they're both on the latest.